prelude/README.md

```diff
@@ -150,7 +150,7 @@ products # groupBy productCategory # filter isInStock # length
 is equivalent to
 
 ```purescript
-length (groupBy productCategory (filter isInStock (products)))
+length (filter isInStock (groupBy productCategory (products)))
```

Comment:
Actually, I think the other one was wrong :) The idea was that `isInStock` applies to a product.